src/Opts.hs

```diff
@@ -61,6 +62,10 @@ parser =
                 ( O.strOption
                     ( O.long "merge-file" <> O.short 'f' <> O.help "Merge a specific file")
                 )
+            <*> O.option O.auto
+                (O.long "context" <> O.metavar "LINECOUNT" <> O.showDefault <> O.value 3
+                    <> O.help "Number of context lines around dumped diffs"
```

Comment:
Note that in `git diff` and `diff`, this is also called `-U`

Reply:
Ah cool, added it now